tests/bluetooth/tester/boards/nrf52_bsim.conf

```diff
@@ -0,0 +1,13 @@
+# CONFIG_TEST enforces minimal logging, which we don't want
```

Comment:
For reviewers: Should we place the BSIM board files in the test instead of the BT Tester directory?